src/agent/CommandStart.ts

```diff
@@ -12,6 +12,7 @@ import type { DeepPartial } from 'polykey/dist/types';
 import type { RecoveryCode } from 'polykey/dist/keys/types';
 import childProcess from 'child_process';
 import process from 'process';
+import * as fs from 'fs';
```

Comment:
This is unnecessary.

Reply:
I think you can do `import fs from 'fs';`.

Reply:
On the other hand I'm pretty sure there's a consistent FS API somewhere. Are we making use of fa injection from the utils @tegefaulkes?

Reply:
I'm not exactly sure what you mean? I think we have an interface we use when a fs needs to be passed into something, but for the top level like this we just import `fs`.

Reply:
This has been fixed.

Reply:
I think I was trying to avoid nodeisms, but since this is PK CLI, that doesn't really matter.